config.json

```diff
@@ -40,5 +43,8 @@
 		"validation_token_expire_seconds": 300,
 		"reset_password_token_expire_seconds": 300,
 		"reset_password_code_length": 8
+	},
+	"socializing": {
+		"date_format": "%Y-%m-%d %H:%M"
```

Comment:
I told you it should be configurable to make users' profile public or private by default, I don't know what is this and how it could be helpful! even if it was the case you are using this date format statically in many files like:
```
resources/user/Socializing/Activity.py 
yml/links_by_user_id.yml
```
therefore please remove this!